composer.json

```diff
@@ -1,16 +1,25 @@
 {
     "name":        "hostnet/asset-bundle",
     "type":        "symfony-bundle",
-    "description": "Bundle to glue Assets",
+    "description": "Hostnet asset bundle allows for configuring the hostnet/asset-lib from within your symfony app",
     "license":     "MIT",
+    "authors": [
+        {
+            "name":  "Yannick de Lange",
+            "email": "ydelange@hostnet.nl"
+        }
+    ],
     "require": {
-        "php":               ">=7.1",
-        "hostnet/asset-lib": "dev-master",
-        "symfony/symfony":   "^3.3"
+        "php":                      "^7.1",
+        "hostnet/asset-lib":        "dev-master",
+        "symfony/framework-bundle": "^3.3",
+        "symfony/http-foundation":  "^2.7|^3.0",
+        "symfony/process":          "^3.3"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.3",
-        "hostnet/phpcs-tool": "^5.2"
+        "hostnet/phpcs-tool": "^5.2",
+        "phpunit/phpunit":    "^6.3",
+        "twig/twig":          "^2.4"
```

Comment:
Is this really the lowest version we can support? The feature set doesn't really change that often for twig, perhaps we can leave this a bit lower?